docker/webui/run_openqa.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# prepare environment
+rm -f /run/dbus/pid
+/usr/share/openqa/script/upgradedb --user geekotest --upgrade_database
+
+# run services
+dbus-daemon --system --fork
+su geekotest -c '/usr/share/openqa/script/openqa-scheduler &'
+su geekotest -c '/usr/share/openqa/script/openqa-websockets &'
+su geekotest -c '/usr/share/openqa/script/openqa gru -m production run &'
+rcapache2 start && su geekotest -c '/usr/share/openqa/script/openqa prefork -m production --proxy'
```

Comment:
Well, I couldn't care less actually and this is probably the wrong
forum for the discussion anyways. I just couldn't resist to comment
when I saw those su calls which really look retarded to my eyes.
Yes, maybe that's docker style and maybe the original idea was
"chroot for a single process" but more complex services just don't
fit into that model. systemd does solve the problem nicely. I'd
consider the overhead of systemd in this case the lesser evil
compared to using sysv style shell hacks :-)